parsers/verilog.c

```diff
@@ -1577,8 +1580,20 @@ static int tagNameList (tokenInfo* token, int c)
 	{
 		c = processType(token, c, &kind);	// update token and kind
 
-		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`')
+		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`' || c == '(')
 		{
+			if(c == '('){
+					token->kind = K_INSTANCE;
+					kind = K_INSTANCE;
+					c = skipPastMatch ("()");
+					if(c != ';'){
```

Comment:
we don't use this style, putting { at the end of line IN THIS FILE.

if (c == '(')
{
   ...

is expected.

Reply:
Thanks, it's my first time to do this job. I will back again

Reply:
This project is looking for persons like you!